### DISCUSSION TITLE
Can you Deploy Remix with ec2 aws ?

### DISCUSSION THREAD

**Discussion by pullmann4rent** [remix-run/remix#8359] [Q&A] [answered]
Hello

I want to use my own backend and I use ec2 now I want to use Remix and I can use my own Adapter but can I deploy Remix on ec2 ? I only see Tutorial on lambda

**Comment by sergiodxa** [comment, marked as answer]
Yes you can, it should work the same way as any other Node.js app, upload your code, install dependencies, run build and then start the app.

Aside of that anything else that you could do with Node it could work with Remix.

And you can use the Remix App Server or the Express adapter, from the officially maintained ones.
